src/web3/networkConfig.ts

```diff
@@ -70,6 +71,9 @@ interface NetworkConfig {
 const CLOUD_FUNCTIONS_STAGING = 'https://api.alfajores.valora.xyz'
 const CLOUD_FUNCTIONS_MAINNET = 'https://api.mainnet.valora.xyz'
 
+const BLOCKCHAIN_API_STAGING = 'https://blockchain-api-dot-celo-mobile-alfajores.appspot.com'
```

Comment:
I appreciate the effort to decouple a test environment from the alfajores testnet, but I don't know if I've used the "staging" environment name elsewhere at Valora. How about "dev" or "development"?

Reply:
I was following the `CLOUD_FUNCTIONS_STAGING` example from above (and elsewhere in this file). The file itself is somewhat inconsistent between `STAGING` and `ALFAJORES`, so I opted for `STAGING` here rather than introducing a new convention.